spec/spec_helper.rb

```diff
@@ -216,6 +216,22 @@ def initialize(current_user, action_name, params)
   end
 end
 
+class PostsView
+  include Pundit::Authorization
+  include Pundit::Helper
+  # Mark protected methods public so they may be called in test
+  # rubocop:disable Style/AccessModifierDeclarations
+  public(*Pundit::Authorization.private_instance_methods)
+  # rubocop:enable Style/AccessModifierDeclarations
```

Comment:
Hi! Sorry about coming in this late.

I don't think we need to make any private instance methods public at all? Also the comment refers to `protected` methods but this does `private` methods.

What about this?

```suggestion
  def self.helper(mod)
    include(mod)
  end

  include Pundit::Authorization
```